### PR TITLE
ENG 1590 Update UI delete workflow documentation

### PR DESCRIPTION
**Pull request by eunice-chan**
Delete workflow can now additionally delete saved objects. Update documentation accordingly.

### CODE REVIEW

**Review by likawind — COMMENTED** [review]
Should we add some screenshots here? Also I'm wondering if we could generate some preview for the updated doc so that it's more intuitive to review the changes @vsreekanti

File: workflows/deleting-a-workflow.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ On the Aqueduct UI, navigate to the settings pane by clicking on the gear icon o
 
 ![](<../.gitbook/assets/image (6) (1).png>)
 
-After workflow deletion has succeeded, you will be redirected to the workflows page.
+You can delete the workflow by following the instructions. You are given the option to delete the entirety of any objects saved by the workflow.
```
Comment (vsreekanti) [inline workflows/deleting-a-workflow.md:15]: I think we could add a bit more detailed here. The word "entirety" is carrying a lot of weight, which we should explain further. I would say two separate things: (1) you can choose which objects to delete; (2) any objects you pick will be completely deleted.

File: workflows/deleting-a-workflow.md [hunk 2]
```diff
@@ -12,7 +12,9 @@ On the Aqueduct UI, navigate to the settings pane by clicking on the gear icon o
 
 ![](<../.gitbook/assets/image (6) (1).png>)
 
-After workflow deletion has succeeded, you will be redirected to the workflows page.
+You can delete the workflow by following the instructions. You are given the option to delete the entirety of any objects saved by the workflow. 
+
+After workflow deletion has succeeded, if you delete any saved objects, you will be shown the results of the deletion (succeeded or failed) and any associated logs. Once you are done, you will be redirected to the workflows page.
```
Comment (vsreekanti) [inline workflows/deleting-a-workflow.md:17]: We should also clarify here that the workflow will be deleted regardless of whether the underlying objects are deleted.

**Review by vsreekanti — CHANGES_REQUESTED** [review]
A couple clarifications. I agree with @likawind that adding a screenshot of the deletion UI would be nice.

File: workflows/deleting-a-workflow.md [hunk 3]
```diff
@@ -12,9 +12,13 @@ On the Aqueduct UI, navigate to the settings pane by clicking on the gear icon o
 
 ![](<../.gitbook/assets/image (6) (1).png>)
 
-You can delete the workflow by following the instructions. You are given the option to delete the entirety of any objects saved by the workflow. 
+After you press the "Delete" button, you will be asked to confirm you wish to delete the workflow. You will also be given a list of objects saved by the workflow. You can select the objects in the list for deletion. Any objects you pick will be completely deleted.
 
-After workflow deletion has succeeded, if you delete any saved objects, you will be shown the results of the deletion (succeeded or failed) and any associated logs. Once you are done, you will be redirected to the workflows page.
+![](<../.gitbook/assets/workflow_deletion_confirmation_page.png>)
+
+After workflow deletion has succeeded, if you delete any saved objects, you will be shown the results of the deletion (succeeded or failed) and any associated logs. Object deletion is done best-effort and the workflow will be deleted regardless of the success of the object deletions. Finally, you will be redirected to the workflows page.
```
Comment (vsreekanti) [inline workflows/deleting-a-workflow.md:19]: The content is good, but the wording is a little confusing. Let's try something like this: 
> Once you confirm deletion of the workflow, Aqueduct will delete the workflow's metadata as well as any of the data objects you selected on the previous screen. Object deletion is done best-effort, and the workflow will be deleted whether or not the associated objects are successfully deleted. After workflow deletion has completed, you will be shown a list of all the objects that Aqueduct attempted to delete and whether those delete operations succeeded or not.

**Review by vsreekanti — APPROVED** [review]
I left one edit inline @eunice-chan, but this is good!

**Pull request merged**